prawn-core.gemspec

```diff
@@ -0,0 +1,45 @@
+dir = File.expand_path(File.dirname(__FILE__))
+
+Gem::Specification.new do |spec|
+  spec.name = 'prawn-core'
+  spec.version = File.read(File.expand_path('VERSION', dir)).strip
+  spec.platform = Gem::Platform::RUBY
+  spec.summary = 'A fast and nimble PDF generator for Ruby'
+
+  spec.authors = [
+    'Gregory Brown',
+    'Guilherme Cassolato'
+  ]
+  spec.email = [
+    'gregory.t.brown@gmail.com',
+    'guicassolato@gmail.com'
```

Comment:
It would be good to use your redhat email and maybe add the 3scale-system-dev one too